README.md

```diff
@@ -108,6 +108,13 @@ These are satRdays organiser groups / locations that have already run a satRday
 
 ## New satRdays in July - September 2020
 ### Europe [0/3 slots filled]
```

Comment:
can you pls also update the number of slots filled in?

Reply:
Sure.